src/components/security_manager/src/ssl_context_impl.cc

```diff
@@ -136,23 +138,33 @@ std::map<std::string, CryptoManagerImpl::SSLContextImpl::BlockSizeGetter>
     CryptoManagerImpl::SSLContextImpl::max_block_sizes =
         CryptoManagerImpl::SSLContextImpl::create_max_block_sizes();
 
+std::string GetName(X509_NAME* x509_name) {
```

Comment:
@dev-gh Right but these changes are critical and needed for further development.